inst/templates/mapping_template_CAMPAIGNers.csv

```diff
@@ -242,8 +242,8 @@ NA;2;carbon management;Carbon Capture|Storage|Electricity;Mt CO2/yr;Carbon Manag
 210;2;emissions (CO2);Emissions|CO2|Energy|Demand|Industry|Pulp and Paper;Mt CO2/yr;;;;;;Fossil CO2 emissions from fuel combustion in industry in the pulp and paper sector;
 211;2;emissions (CO2);Emissions|CO2|Energy|Demand|Industry|Non-Ferrous Metals;Mt CO2/yr;;;;;;Fossil CO2 emissions from fuel combustion in industry in the non-ferrous metals sector;
 212;2;emissions (CO2);Emissions|CO2|Energy|Demand|Other Sector;Mt CO2/yr;;;;;;CO2 emissions from fuel combustion in other energy supply sectors (please provide a definition of other sources in this category in the 'comments' tab);
-213;2;emissions (CO2);Emissions|CO2|Energy|Demand|Residential;Mt CO2/yr;;;;not modelled but post-processed;;Fossil CO2 emissions from fuel combustion in residential (IPCC category 1A4b);
```

Comment:
do you want to delete this comment as well?

Reply:
That was a mistake, thank you for pointing this out!